_squirtle/mod.ts

```diff
@@ -300,28 +302,29 @@ const runCheck = async (argv: any) => {
   const siteUrl = isDev ? 'https://quest-dev.stellar.org' : 'https://quest.stellar.org'
 
   if (!user.pk) {
-    const missingPkConfirmed = await Confirm.prompt(`You have not yet connected your Stellar wallet.
-   This will affect your ability to claim NFT and XLM rewards.
-   Would you like to connect your Stellar wallet?`);
+    const missingPkConfirmed = await Confirm.prompt(MISSING_PK_MESSAGE);
 
     if (missingPkConfirmed) {
       return browse(`${siteUrl}/settings`)
     }
   }
 
   else if (
-    !(
-      user.taxStatus === 'accepted'
-      || user.taxStatus === 'pending'
-      || user.taxStatus === 'completed'
-      || (user.taxStatus === 'requested' && user.tax)
-    ) || !(
-      (user.kyc.ofac ? user.kyc.ofac.replace(/\W/g, '_') === 'no_match' : true)
-      && user.kyc.status === 'approved'
+    SHOW_TAX_KYC
+    && (
+      !(
+        user.taxStatus === 'accepted'
+        || user.taxStatus === 'pending'
+        || user.taxStatus === 'completed'
+        || (user.taxStatus === 'requested' && user.tax)
+      ) || !(
+        (user.kyc.ofac ? user.kyc.ofac.replace(/\W/g, '_') === 'no_match' : true)
+        && user.kyc.status === 'approved'
+      )
     )
   ) {
     const missingPkConfirmed = await Confirm.prompt(`You have not yet completed the KYC flow and/or uploaded your tax documents.
-   This will affect your ability to claim XLM rewards.
+   This will affect your ability to claim XLM awards.
```

Comment:
Is it actually award vs reward? I thought it was reward. Now I'm second guessing myself 😳

Reply:
okok you're right. Feel free to merge, build, deploy, etc and ensure it's all working. Thanks for this!